src/stylesheets/components/_dropdown.scss

```diff
@@ -1,55 +1,27 @@
-.has-js .dropdown {
-  &:not(.open) {
-    .dropdown__toggle {
-      margin-bottom: 0;
-    }
-    .dropdown__content {
-      @extend .visually-hidden;
-      padding-top: 0;
-      padding-bottom: 0;
-    }
-  }
-}
-
 .dropdown {
   background-color: $grey-4;
 }
 
-.dropdown__toggle {
-  position: relative;
-  @include headings-margin-top(0);
-  @include last-child;
-}
-
-.dropdown__content {
-  font-weight: 400;
-  border: solid $grey-3;
-  border-width: 1px 0 0;
-  opacity: 1;
-  height: auto;
-  overflow: hidden;
-  transition: all 0.1s linear;
+.dropdown__details {
+  border-top: 1px solid $grey-3;
   padding: $base-spacing-unit-small;
   @include last-child;
+  transition: padding-top 0.1s linear, padding-bottom 0.1s linear;
 }
 
-/*--- Component Rules ---*/
+.has-js .dropdown__details:not(.open) {
```

Comment:
Can you verify that this is required. There is already `.has-js [data-toggle="content"]:not(.open) { padding: 0; }` that is doing the same thing